assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/config.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""PyFunc Config."""
+
+from enum import Enum
+
+from azureml.model.mgmt.config import _CustomEnum
+
+
+class SupportedVisionTasks(_CustomEnum):
```

Comment:
is it irrelevant class? Since, you are refactoring, can you please check this also.l

Reply:
Using this to decide if we should use mmlabdetection specific model convertor. Will keep it as is.